### PR TITLE
[INLONG-10338][Sort] Fix Sqlserver AuditOperator not serialized

### LINKED ISSUE #10338: [Bug][Sort] Sqlserver connector's AuditOperator was not serialized
### What happened

When I run sqlserver connector, There are some problem as follow.
企业微信截图_93eed270-ec7d-4b1b-952f-c6247868242a

### What you expected to happen

AuditOperator can send audit infotmation to pulsar

### How to reproduce

Move the AuditOperator serialization to correct place.

### Environment

docker

### InLong version

master

### InLong Component

InLong Sort

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by XiaoYou201**
### [INLONG-10338][Sort] Fix AuditOperator not serialized

Fixes #10338 

### Motivation

Make AuditOperator serialized before report audit information.

### Modifications

Some deserialization Code.

### Verifying this change
image

### CODE REVIEW

File: inlong-sort/sort-flink/sort-flink-v1.15/sort-connectors/sqlserver-cdc/src/main/java/org/apache/inlong/sort/sqlserver/RowDataDebeziumDeserializeSchema.java [hunk 1]
```diff
@@ -126,7 +131,17 @@ public static Builder newBuilder() {
...
         this.validator = checkNotNull(validator);
         this.changelogMode = checkNotNull(changelogMode);
-        this.sourceMetricData = sourceMetricData;
+        this.metricOption = metricOption;
+    }
+
+    @Override
+    public void open() {
+        if (metricOption != null) {
+            LOG.info("SourceMetricData create -----------");
```
Comment (EMsnap) [inline inlong-sort/sort-flink/sort-flink-v1.15/sort-connectors/sqlserver-cdc/src/main/java/org/apache/inlong/sort/sqlserver/RowDataDebeziumDeserializeSchema.java:140]: please keep log format consistent with other places

File: inlong-sort/sort-flink/sort-flink-v1.15/sort-connectors/sqlserver-cdc/src/main/java/org/apache/inlong/sort/sqlserver/DebeziumChangeFetcher.java [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+ * A Handler that convert change messages from {@link DebeziumEngine} to data in Flink. Considering
+ * Debezium in different mode has different strategies to hold the lock, e.g. snapshot, the handler
+ * also needs different strategy. In snapshot phase, the handler needs to hold the lock until the
+ * snapshot finishes. But in non-snapshot phase, the handler only needs to hold the lock when
+ * emitting the records.
+ *
+ * @param <T> The type of elements produced by the handler.
+ */
+@Internal
+public class DebeziumChangeFetcher<T> {
```
Comment (EMsnap) [inline inlong-sort/sort-flink/sort-flink-v1.15/sort-connectors/sqlserver-cdc/src/main/java/org/apache/inlong/sort/sqlserver/DebeziumChangeFetcher.java:53]: Add description if copied

File: inlong-sort/sort-flink/sort-flink-v1.15/sort-connectors/sqlserver-cdc/src/main/java/org/apache/inlong/sort/sqlserver/RowDataDebeziumDeserializeSchema.java (hunk 1 above)
Comment (XiaoYou201) [reply inlong-sort/sort-flink/sort-flink-v1.15/sort-connectors/sqlserver-cdc/src/main/java/org/apache/inlong/sort/sqlserver/RowDataDebeziumDeserializeSchema.java]: fix it，thx～

File: inlong-sort/sort-flink/sort-flink-v1.15/sort-connectors/sqlserver-cdc/src/main/java/org/apache/inlong/sort/sqlserver/DebeziumChangeFetcher.java (hunk 2 above)
Comment (XiaoYou201) [reply inlong-sort/sort-flink/sort-flink-v1.15/sort-connectors/sqlserver-cdc/src/main/java/org/apache/inlong/sort/sqlserver/DebeziumChangeFetcher.java:55]: fix it， thx～

**Review by aloyszhang — APPROVED** [review]

**Review by EMsnap — APPROVED** [review]

**Pull request merged**